### ISSUE TITLE
libmavconn: Support to setup packet signing (mavlink 2.0)

### ISSUE THREAD

**Issue by vooon** [mavlink/mavros#545]

**Comment by TSC21** [comment]
Closed as solved

**Comment by vooon** [comment]
@TSC21 no, v2.0 **signing** is not yet supported (v2.0 has feature to prevent hacking your drone via unencrypted telemetry channels by adding crypto hash).

**Comment by TSC21** [comment]
We should keep track of https://github.com/PX4/Firmware/pull/7530 to this same matter.

**Comment by ivanzivotic** [comment]
@vooon Is there some plan for implementing mavlink 2.0 packet signing feature, either for ROS1 or ROS2? Thanks

**Comment by vooon** [comment]
No.
